src/Compilers/Core/Portable/Compilation/ParseOptions.cs

```diff
@@ -127,9 +129,9 @@ public ParseOptions WithFeatures(IEnumerable<KeyValuePair<string, string>> featu
         /// </summary>
         public abstract IEnumerable<string> PreprocessorSymbolNames { get; }
```

Comment:
The implementation of EqualsHelper implies this can be null?

Reply:
Looked at the C# and VB sources here and they both have this as non-null values as best I can tell. Seems the `null` check is just being a bit careful here. 

---
In reply to: [360010135](https://github.com/dotnet/roslyn/pull/40493#discussion_r360010135) [](ancestors = 360010135)